### DISCUSSION TITLE
Error message in GUI login  "Login failed. Please try again"

### DISCUSSION THREAD

**Discussion by augustobr1to** [tteck/Proxmox#3441] [Help Needed] [answered]
Hello guys, something strange happened with my home lab proxmox. Today, when I tried to log in with GUI, i received an error message "
Login failed. Please try again"

I saw this post https://forum.proxmox.com/threads/login-failed-but-logs-in-anyway.146754/ on official proxmox forum talking about post-install script.

Anyone can help me?

**Comment by tteck** [comment]
Read the entire thread! 
> I was complaining on discord about that message every time when you login and you dont have a valid license.
**I gave one of my friends full access, and he fixed it, after that the issues started**

https://forum.proxmox.com/threads/login-failed-but-logs-in-anyway.146754/post-664696 
Nothing in this repository can cause such issues!

**Reply by augustobr1to** [reply]
Thanks for the fast response... yeah...  I didn't know what happened with the GUI login...  SSH works fine...

**Comment by tteck** [comment]
What Realm are you using?

![Screenshot 2024-07-29 8 15 38 AM](https://github.com/user-attachments/assets/6ce74c1c-1e94-487b-b5c5-55855991cda8)

**Reply by augustobr1to** [reply]
![image](https://github.com/user-attachments/assets/ec5bc683-b7f8-46d3-b650-6d4969588c59)

**Reply by tteck** [reply]
What is (+ oath)?

**Reply by augustobr1to** [reply]
> What is (+ oath)?

There are only two options:

- Linux PAM standard authentication (+ oath)
- Proxmox VE authentication server

![image](https://github.com/user-attachments/assets/8bcd0ac3-a652-48f7-997d-90d683a37615)

**Reply by tteck** [reply]
Realm Enforced Two-Factor Authentication?

**Reply by augustobr1to** [reply]
Yes, I had TFA, but reading the official forum I removed TFA because I had the same error. Even after removing TFA the error persists.

I tried to create another user, but the error remains the same.

**Reply by tteck** [reply, marked as answer]
> users will be locked out after too many failed 2nd factor login attempts.

https://pve.proxmox.com/pve-docs/pve-admin-guide.html#pveum_tfa_auth
I would think since it still shows `(+ oath)` TFA is not disabled.

**Reply by augustobr1to** [reply]
I will try to disable enforce TFA

**Reply by augustobr1to** [reply]
@tteck It works! Thanks man!! 
![image](https://github.com/user-attachments/assets/a63b2c32-5a0b-469e-ba6e-572b79f5c09a)
